ai/coach.py

```diff
@@ -79,8 +79,16 @@ def wait_for_geometry(self):
         self.game_state.const = self.field
         self.logger.debug('Geometry received from the Engine in {:0.2f} seconds.'.format(time() - start))
 
+    def wait_for_referee(self):
+        self.logger.debug('Waiting for commands from the referee')
+        while self.referee_queue.qsize() == 0:
+            self.logger.debug('Referee is not active or port is set incorrectly (should be 20011)')
```

Comment:
Tu peux spécifier le port courant, l'objet Config() contient ce genre d'information:
'Referee is not active or port is incorrect (current port is {})'.format(Config()['COMMUNICATION']['referee_port'])

Reply:
Faudrait faire cela juste avec le flag competition mode.